### PR TITLE
Add support for passing metadata to bk local run.

### PR DESCRIPTION
**Pull request by stefan-improbable**

### CODE REVIEW

File: cmd/bk/main.go [hunk 1]
```diff
@@ -310,12 +310,19 @@ func run(args []string, exit func(int)) {
 			Short('E').
 			StringsVar(&runCmdCtx.Env)
 
+		cmd.
+			Flag("metadata", "Metadata to pass to the agent").
```
Comment (lox) [inline cmd/bk/main.go]:
```suggestion
			Flag("meta-data", "Meta-data to pass to the build").
```

**Comment by lox** [comment]
Seems legit! Made some minor feedback.

**Comment by stefan-improbable** [comment]
> Seems legit! Made some minor feedback.

Thanks, I've accepted that suggestion.

**Comment by lox** [comment]
Thanks very much for the contribution @stefan-improbable!

**Comment by stefan-improbable** [comment]
> Thanks very much for the contribution @stefan-improbable!

@lox thanxs for merging this.

**Pull request merged**